src/rulesets/path.ts

```diff
@@ -1,9 +1,25 @@
-const URL = require('url').URL;
-import { dirname, resolve } from 'path';
+import * as path from '@stoplight/path';
+import { PROJECT_ROOT } from '../consts';
+import { isURL } from '../fs/reader';
+import { rulesetsMap } from './map';
 
 export function resolvePath(from: string, to: string) {
-  if (from.startsWith('http')) {
-    return new URL(to, from).href;
+  const mapped = rulesetsMap.get(to);
+  if (mapped !== void 0) {
+    to = mapped;
   }
-  return resolve(dirname(resolve(process.cwd(), from)), to);
+
+  if (isURL(to) || path.isAbsolute(to)) {
+    return to;
+  }
+
+  if (to.startsWith('@stoplight/spectral/')) {
+    try {
+      return to.replace('@stoplight/spectral/', require.resolve('@stoplight/spectral'));
```

Comment:
From looking at marcs comments I thought we were going to support any NPM path, so peoplecould ship their rulesets as NPM packages or something. This looks like we're just pretending and using the syntax to enable our own? Is this to save time before implementing the feature entirely after 4.0 is released?

Reply:
> From looking at marcs comments I thought we were going to support any NPM path, so peoplecould ship their rulesets as NPM packages or something. This looks like we're just pretending and using the syntax to enable our own? Is this to save time before implementing the feature entirely after 4.0 is released?


True, I can add support for external npm modules too. It's my oversight. Thanks for spotting.